vars/common.yml

```diff
@@ -2,11 +2,26 @@
 aws_region: eu-west-1
 
 api:
-  rds_port: 5432
-  db_user: "digitalmarketplace"
-  db_name: "digitalmarketplace_api"
-  db_instance_type: db.m1.small
-  db_allocated_storage: 5
+  instance_type: t2.micro
+  min_instance_count: 1
+  max_instance_count: 2
+
+search_api:
+  instance_type: t2.micro
+  min_instance_count: 1
+  max_instance_count: 2
+
+admin_frontend:
+  instance_type: t2.micro
+  min_instance_count: 1
+  max_instance_count: 2
```

Comment:
Yes, but I think we should do this once we have the production account and know the final instance sizes that we'll use